projects/micro-dash/src/lib/lang/to-string.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Converts `value` to a string. An empty string is returned for `null` and `undefined` values.
+ *
+ * Differences:
+ * - We don't handle -0 as a special case
```

Comment:
I added this as a high-level difference in the readme. You can remove this line now.